src/seedu/addressbook/commands/Command.java

```diff
@@ -37,6 +37,14 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     }
 
     /**
+     *
+     * @return if the (child) class will mutate the data
```

Comment:
`@return true if the command potentially mutates the data`? 
* mention `true`
* mention potentially
* no need to mention child class